core/src/main/resources/defaults/blueprints/7.2.8/cdp-data-engineering-ha.bp

```diff
@@ -365,7 +365,13 @@
           {
             "refName": "tez-GATEWAY-BASE",
             "roleType": "GATEWAY",
-            "base": true
+            "base": true,
+            "configs": [
+             {
+               "name": "tez-conf/tez-site.xml_client_config_safety_valve",
+               "value": "<property><name>tez.runtime.pipelined.sorter.lazy-allocate.memory</name><value>true</value></property>"
+             }
+           ]
```

Comment:
Could you please correct the indentation? Also applies to all other files below.

Reply:
Indentation changes made in all the files.